docs/csharp/programming-guide/unsafe-code-pointers/how-to-access-an-array-element-with-a-pointer.md

```diff
@@ -38,7 +38,7 @@ Notice that the expression `charPointer[i]` is equivalent to the expression `*(c
 ## See also
 
 - [C# Programming Guide](../../../csharp/programming-guide/index.md)
-- [Pointer Expressions](../../../csharp/programming-guide/unsafe-code-pointers/pointer-expressions.md)
+- [Pointer Expressions](../../../csharp/programming-guide/unsafe-code-pointers/how-to-obtain-the-value-of-a-pointer-variable.md)
```

Comment:
Should we simply remove this link instead? We no longer have the Pointer Expressions topic so we no longer need to link to the parent topic. Same feedback applies to the next articles too.